### PR TITLE
Pgp support

### PR DESCRIPTION
**Pull request by mahmoudimus**
## Description of changes in release / Impact of release:

This is an up-to-date PR from the work done by #168. 

## Documentation

 TODO

### CODE REVIEW

File: larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/dto/TextPIIEntity.java [hunk 1]
```diff
@@ -95,19 +92,13 @@ public boolean isImmutable() {
...
-
-    final Dict.Builder<String, Object> builder = Dict.builder();
-    final List<String> keySet = new ArrayList<>(mapRepresentation.keySet());
-    Collections.sort(keySet);
-    for (String k: keySet) {
-      builder.put(k, mapRepresentation.get(k));
-    }
-
-    return builder.buildImmutable();
+    return Dict.<String, Object>builder()
```
Comment (mahmoudimus) [inline larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/dto/TextPIIEntity.java:95]: @Agitolyev wanted to check to see if you're OK w/ this re-factor.

File: larky/src/main/resources/stdlib/bz2.star [hunk 2]
```diff
@@ -0,0 +1,395 @@
+"""
```
Comment (mahmoudimus) [inline larky/src/main/resources/stdlib/bz2.star:1]: plz ignore this file, it does not work yet and won't anytime soon.

File: larky/src/main/resources/vendor/Crypto/PublicKey/DSA.star [hunk 3]
```diff
@@ -1,207 +1,685 @@
-def DsaKey(object):
-    """
+# -*- coding: utf-8 -*-
```
Comment (mahmoudimus) [inline larky/src/main/resources/vendor/Crypto/PublicKey/DSA.star:1]: @aslepakurov we will probably have a merge conflict here but that's ok.

**Comment by aslepakurov** [comment]
Where are all files that are being used in tests? Also tests use `open()` functions.
I wanted to rewrite it to `load()` but still there is a question of files. 
Please advice @mahmoudimus

**Comment by mahmoudimus** [comment]
> Where are all files that are being used in tests? Also tests use open() functions.
> I wanted to rewrite it to load() but still there is a question of files.
> Please advice @mahmoudimus

Please see https://github.com/verygoodsecurity/starlarky/blob/3dee888bdc3867d4587aa33e45b811b2d23a8d16/larky/src/test/resources/vendor_tests/OpenPGP/data_test_fixtures.star

**Pull request merged**